src/drivers/driver_framework_wrapper/df_ltc2946_wrapper/df_ltc2946_wrapper.cpp

```diff
@@ -70,9 +69,7 @@ class DfLtc2946Wrapper : public LTC2946
 private:
 	int _publish(const struct ltc2946_sensor_data &data);
 
-	orb_advert_t	_battery_pub{nullptr};
-	battery_status_s _battery_status{};
-	Battery _battery{};
+	Battery1 _battery{};
```

Comment:
It seems a bit weird to have the instance explicitly in the type like this. Would a generic Battery<> or AnalogBattery<> work?